### DISCUSSION TITLE
How save the outputs from LLMs as it shown in the logs.

### DISCUSSION THREAD

**Discussion by innocent-charles** [explosion/spacy-llm#381] [Q&A]
Hello ! can anyone help me with how save the outputs of LLMs as it shown in the logs but a little better like json with entities and value extracted.

**Comment by rmitsch** [comment]
The output (i. e. prompts and responses) are available in `Doc.user_data["llm_io"]`.

Unfortunately there is a mistake in the [docs](https://spacy.io/api/large-language-models#init) here at the moment:
> Whether to save LLM I/O (prompts and responses) in the `Doc._.llm_io` custom attribute. 

`Doc._.llm_io` should actually be `Doc.user_data["llm_io"]`. We'll fix the docs.

**Comment by innocent-charles** [comment]
Thanks much a lot @rmitsch
